azerothcore/azerothcore-wotlk
# Add Black Bear Pack northwest of the Ironforge Airfield.

## Changes Proposed:
-  Adds Black Bear and two Young Black Bears in a small area northwest of the Ironforge airfield.

## Issues Addressed:

- Makes progress towards #6994 

## SOURCE:

TBCC Sniffs (Partially)

## Tests Performed:

- Observed that the creatures correctly spawned in the area.


## How to Test the Changes:


1. Run the query
2. Use the in-game command .go xyz -4322 -1464 415
3. Observe the three bears spawned

## Known Issues and TODO List:


- [x] Still have to find the wander_distance of the two young black bears and their actual spawn position


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
